README.md

```diff
@@ -1,18 +1,98 @@
-## Chess
+<h1 align='center'>Chess</h1>
 
-Building a platform where people can
+## Table of contents
 
-1. Sign up
-2. Create a new match/get connected to an existing match
-3. During the match, let users play moves
-4. Have a rating system that goes up and down similar to standard chess rating
+- [Prerequisites](#prerequisites)
+- [Setup](#setup)
+  - [Using Docker](#using-docker)
+  - [Without Docker](#without-docker)
+- [Contributing](#contributing)
 
-## Tech stack
+## Prerequisites
 
-Let's keep it simple
+Before you begin, ensure you have met the following requirements
 
-1. React for Frontend
-2. Node.js for Backend
-3. Typescript as the language
-4. Separate Websocket servers for handling real time games
-5. Redis for storing all moves of a game in a queue
+- Node.js and npm installed on your machine.
+- Docker and docker-compose installed (optional, required for Docker setup).
+- Access to a PostgreSQL database (can be local or hosted).
+- Google OAuth Client Credentials
+
+## Setup
+
+### Using Docker
+
+1. Clone the repository:
+   ```bash
+   git clone https://github.com/code100x/chess.git
+   ```
+2. Navigate to the project directory:
+   ```bash
+   cd chess
+   ```
+3. Copy the `.env.example` in
+   `/apps/frontend`
+   `/apps/backend`
+   `/packages/db`
+   to `.env` in the same directory
+4. Replace the Google Client ID and Secret in the `/apps/backend/.env` file.
+5. Update the DatabaseURL in the `/packages/db/.env`
+    ```bash
+   DATABASE_URL="postgresql://postgres:chess100x@172.20.0.8:5432/chess100x"
+    ```
+6. Run the following command in the root dir to launch all the services:
+   ```bash
+   docker-compose up
+   ```
+
+### Without Docker
+
+1. clone the repository:
+   ```bash
+   git clone https://github.com/code100x/chess.git
+   ```
+2. Navigate to the project directory:
+   ```bash
+   cd chess
+   ```
+3. (optional) Start a PostgreSQL database using Docker:
+   ```bash
+   docker run -d \
+       --name chess100x \
+       -e POSTGRES_USER=postgres \
+       -e POSTGRES_PASSWORD=chess100x \
+       -e POSTGRES_DB=chess100x \
+       -p 5432:5432 \
+       postgres
+   ```
```

Comment:
Why does without docker instructions require docker?

Reply:
Will update this clearly